package.json

```diff
@@ -53,58 +35,53 @@
     "!**/*.d.ts"
   ],
   "dependencies": {
-    "cliui": "^8.0.1",
+    "cliui": "^9.0.1",
     "escalade": "^3.1.1",
     "get-caller-file": "^2.0.5",
     "require-directory": "^2.1.1",
-    "string-width": "^4.2.3",
+    "string-width": "^7.2.0",
     "y18n": "^5.0.5",
     "yargs-parser": "^21.1.1"
   },
   "devDependencies": {
-    "@rollup/plugin-terser": "^0.4.4",
-    "@rollup/plugin-typescript": "^12.1.2",
+    "@babel/eslint-parser": "^7.26.10",
+    "@babel/preset-typescript": "^7.26.0",
     "@types/chai": "^4.2.11",
     "@types/mocha": "^9.0.0",
     "@types/node": "^20.0.0",
+    "@typescript-eslint/eslint-plugin": "^8.26.1",
     "browserslist-generator": "^2.0.1",
     "c8": "^9.1.0",
     "chai": "^4.2.0",
     "chalk": "^4.0.0",
     "concurrently": "^7.6.0",
     "cpr": "^3.0.1",
     "cross-env": "^7.0.2",
-    "cross-spawn": "^7.0.0",
-    "eslint": "^8.56.0",
+    "cross-spawn": "^7.0.6",
+    "eslint": "^8.57.1",
     "eslint-plugin-prettier": "^5.1.2",
     "gts": "^5.2.0",
     "hashish": "0.0.4",
-    "mocha": "^9.0.0",
+    "mocha": "^10.8.2",
     "rimraf": "^3.0.2",
-    "rollup": "^3.29.4",
-    "rollup-plugin-cleanup": "^3.2.1",
     "typescript": "^5.7.2",
```

Comment:
Asking a question rather than recommending as such. TypeScript does not follow semver, and the yargs and yargs-parser builds have broken in past for minor versions. We might use "~5.7.2" or even "5.7.2", and only apply upgrades after `renovate` tries them?

Reply:
It's been long while since TypeScript did this to us, and I guess I have a short memory 😆 

I'm tempted to leave it as is for now. We can revisit the decision in the future if our builds break, and I can be blamed.